eng/Version.Details.xml

```diff
@@ -1,16 +1,16 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.5.24229.2" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="9.0.100-preview.5.24253.16" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>d301a122c4f9d017cdc3a506f831e2edfc124ea7</Sha>
+      <Sha>fa261b952d702c6bd604728fcbdb58ac071a22b1</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.4.24223.11" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.4.24251.3" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>d92ac1f892a7f9c00561db6541671f25f3972eed</Sha>
+      <Sha>4326eb7ed4d03f30ce4a4de1eb028ee76fdaaa3c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.5.272">
+    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.5.274">
       <Uri>https://github.com/xamarin/xamarin-android</Uri>
-      <Sha>78d42197bb95af5ab7c2976e4abef822e8711c58</Sha>
+      <Sha>0781fac6d9e9703eb30c040b24a474ea0a1fd536</Sha>
```

Comment:
If this is the diff in xamarin-android:
* https://github.com/xamarin/xamarin-android/compare/78d42197bb...0781fac6d9

It has no changes to dotnet/runtime or dotnet/emsdk.

Was there some other manual change to those in the `net9.0` branch?